docs/python.md

```diff
@@ -103,7 +103,7 @@ The `Input()` function takes these keyword arguments:
 
 Each parameter of the `predict()` method must be annotated with a type like `str`, `int`, `float`, `bool`, etc. See [Input and output types](#input-and-output-types) for the full list of supported types.
 
-## `Output(BaseModel)`
+## Output
```

Comment:
I was on the fence about these headings when first writing this doc, but I like the simplicity of the change you've made here. Maybe we should we do this for `Input(**kwargs)` -> `Input` too?

Reply:
I think it's fine how it is. This means "Output" as in, the concept of output. `Input(**kwargs)` is about the Python function `Input()`.

Maybe the structure of the document could clarify this. It's part explaining, part API reference.